ONSdigital/eq-questionnaire-runner
# Updating makefile commands

### What is the context of this PR?
Added more bespoke commands to the Makefile to streamline development processes

### How to review
- Run `make validate-test-schema` to ensure the single schema is validated
- Run `make test-single-functional` & `make test-suite-functional:`to ensure the specified single spec/suite is run
- Run `make test-lint-js` to ensure both JS lint and JS tests are run
- Run `make test-lint-python` to ensure both Python lint and Python tests are run
- Run `make test-functional-headless` to ensure functional tests are run headless

### Checklist
